File: src-py/calcfromradius.py
"""
ST31 Kotlin文法 Lesson1 Ex4

Python実行ファイル。

@author Shinzo SAITO
"""
from calccircleandball import CalcCircleAndBall

rList = [1.0, 3.0, 5.0, 7.0, 9.0]
for radius in rList:
	calc = CalcCircleAndBall(radius)
	print(f"■■■ 半径{radius}の計算結果")
	print(f"円周: {calc.get_circle()}")
	print(f" | 円面積: {calc.get_area()}")
	print(f" | 球面積: {calc.get_surface()}")
	print(f" | 球体積: {calc.get_volume()}")

File: src-js/calcFromRadius.mjs
/**
 * ST31 Kotlin文法 Lesson1 Ex4
 * 
 * JavaScript実行ファイル。
 * 
 * @author Shinzo SAITO
 */
import CalcCircleAndBall from "./CalcCircleAndBall.mjs";

const rList = [1, 3, 5, 7, 9];
for(const radius of rList) {
	const calc = new CalcCircleAndBall(radius);
	console.log(`■■■ 半径${radius}の計算結果`);
	console.log(`円周: ${calc.getCircle()}`);
	console.log(` | 円面積: ${calc.getArea()}`);
	console.log(` | 円面積: ${calc.getSurface()}`);
	console.log(` | 円面積: ${calc.getVolume()}`);
}
